src/targets/services/cliskTimeout.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * This service stops the given job with a 'context deadline exceeded' error. It may be called when
+ * the execution of the clisk konnector is too long or when the flagship app has crashed.
+ * The flagship app will also timeout the job if it restarts but this way, the user will not see
+ * the job as running even outside the flagship app
+ *
+ * This service is supposed to be called directly or via a @in or @at trigger
+ *
+ * Example to call this service :
+ *
+ *
+ *  await client.collection('io.cozy.jobs').create(
+ *     'service',
+ *     {
+ *       name: 'cliskTimeout',
+ *       slug: 'home',
+ *       fields: {
+ *        cliskJobId: '<the job id>'
+ *       }
+ *     }
+ *   )
+ */
+import CozyClient from 'cozy-client'
+import logger from 'cozy-logger'
+import polyfillFetch from './polyfillFetch'
+
+const log = logger.namespace('cliskTimeout')
+
+polyfillFetch()
+
+const TIMEOUT_ERROR = 'context deadline exceeded' // same error as cozy-stack when interrupting a job for timeout
+const JOBS_DOCTYPE = 'io.cozy.jobs'
+
+const main = async () => {
+  const client = CozyClient.fromEnv()
+
+  const { cliskJobId } = getServiceFields()
+
+  if (!cliskJobId) {
+    throw new Error('No cliskJobId field defined. Could find any job to update')
+  }
+
+  await client.save({
+    _id: cliskJobId,
+    _rev: true, // pass a revision to force client.save to call job.update and not job.create. This avoids an unnecessary GET of the job
```

Comment:
I'm not confident about that. The `_rev` is used by Couch to know about history and conflict. @paultranvan what are your thought about that?

Reply:
If you want to update an existing doc, you should indeed pass the actual `_rev` to avoid conflict. The only exception I know is for bulk_update where you can force your own revision, which is used for the cozy sharing. See https://docs.couchdb.org/en/stable/api/database/bulk-api.html#db-bulk-docs and https://github.com/cozy/cozy-stack/blob/52760b3f2908d118d16fc857c57d98c948e47385/pkg/couchdb/bulk.go#L371
I don't think I ever saw `_rev: true`, I'm curious where does it come from @doubleface ?

Reply:
@paultranvan this is just to bypass cozy-client protection on the save https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L792 

@doubleface we should request the io.cozy.jobs to get the latest revision and pass it to the save method. OK this is one more query but we'll avoid a lot of conflict.

Reply:
If we really want to avoid a query to get the rev, we could also pass a `cliskJobRev` alongside the `cliskJobId` (only if we are sure the doc won't be updated in the meantime).